src/views/protected/TransitionViews/TransitionTimeHelp.tsx

```diff
@@ -22,9 +23,9 @@ const styles: object = {
   paper: {
     position: "absolute",
     width: "67%",
-    backgroundColor: theme.palette.background.paper,
-    boxShadow: theme.shadows[5],
-    padding: theme.spacing.unit * 4,
+    //backgroundColor: theme.palette.background.paper,
```

Comment:
You had to comment these because theme was removed

Reply:
@sriharrsha @WillNock I think this was me who removed them...but I changed the backgroundColor to 'white' and changed padding to '1em' and got rid of the boxShadow to make this more consistent with how we've done the other styles. That's how it is in develop now